### PR TITLE
Fix the Makefile for dialyzer

### PR DESCRIPTION
**Pull request by kostis**
- add 'tools' application to the PLT (so that cover functions are known)
 - 'make distclean' should also remove the PLT

### CODE REVIEW

**Comment by aronisstav** [comment]
The "failed build" is because I stopped the duplicate Travis job.

File: Makefile [hunk 1]
```diff
@@ -167,7 +167,7 @@ clean:
 	$(RM) -r ebin
 
 .PHONY: distclean
-distclean: clean
```
Comment (aronisstav) [inline Makefile]: I disagree. I took a look at the following guidelines and I think the current split between `distclean` and `maintainer-clean` is more reasonable.

https://www.gnu.org/prep/standards/html_node/Standard-Targets.html

What do you think?

Comment (kostis) [reply Makefile]: OK.  I can revert this.

**Comment by kostis** [comment]
We can drop this PR and delete the branch.

**Comment by aronisstav** [comment]
Done!

**Pull request closed without merging**